docs/src/man/tour.md

```diff
@@ -7,84 +7,97 @@ CurrentModule = LazySets
 LazySets is a library for set-based computations in Euclidean space.
 The library offers both concrete and lazy set representations,
 where the latter stands for the ability to delay set computations until they are
-needed. The choice of the programming language [Julia](https://julialang.org/) and the accompanying documentation
+needed. The choice of the programming language [Julia](https://julialang.org/)
+and the accompanying documentation
 allow researchers to easily translate set-based algorithms
-from mathematics to software in a platform-independent way, while achieving runtime
-performance that is comparable to statically compiled languages.
+from mathematics to software in a platform-independent way, while achieving
+runtime performance that is comparable to statically compiled languages.
 Combining lazy operations in high dimensions and explicit computations in low
 dimensions, LazySets can be applied to solve complex, large-scale problems.
 The library can handle the most common operations between sets, including
 [Minkowski sum](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/MinkowskiSum/),
 [Cartesian product](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/CartesianProduct/),
-[convex hull](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/ConvexHull/), [intersection](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/Intersection/) and
-[union](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/UnionSet/).
+[convex hull](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/ConvexHull/),
+[intersection](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/Intersection/)
+and [union](https://juliareach.github.io/LazySets.jl/dev/lib/lazy_operations/UnionSet/).
 
-The puropose of this section is to give a high-level overview of the libary and
+The purpose of this section is to give a high-level overview of the library and
 several examples to illustrate basic functionality. Further sections delve into
 more advanced uses.
 
+
 ## Creating sets
 
 Sets can be created using the constructor for each set type. Here we define the
-two-dimensional halfspace
+two-dimensional half-space
 
 ```math
-H = \{ (x, y) ∈ \mathbb{R}^2 : x ≤ y\}.
+H = \{ (x, y) ∈ \mathbb{R}^2 : x ≤ y\} = \{ (x, y) ∈ \mathbb{R}^2 : x - y ≤ 0\}.
```

Comment:
I added this alternative because it corresponds to the `HalfSpace` representation.